setup_dependencies.py

```diff
@@ -170,5 +170,36 @@
                     'WMCore.Services.pycurl_manager'],
         'statics': ['src/couchapps/ACDC+',
                     'src/couchapps/GroupUser+']
+    },
+    't0': {
+        'packages': ['WMCore.Agent+', 'WMCore.Algorithms+',
+                     'WMCore.JobStateMachine', 'WMComponent+',
+                     'WMCore.ThreadPool', 'WMCore.WorkerThreads',
+                     'WMCore.BossAir+', 'WMCore.Credential',
+                     'WMCore.JobSplitting+', 'WMCore.ProcessPool',
+                     'WMCore.Services+', 'WMCore.WMSpec+',
+                     'WMCore.WMBS+', 'WMCore.ResourceControl+',
+                     'WMCore.DataStructs+', 'WMCore.ReqMgr+',
+                     'Controllers+', 'WMQuality.Emulators+',
+                     'WMCore.ACDC', 'Utils'],
+        'modules': ['WMCore.Configuration',
+                    'WMCore.DAOFactory',
+                    'WMCore.WMException',
+                    'WMCore.Lexicon',
+                    'WMCore.WMBS.File'],
+        'systems': ['wmc-web', 'wmc-database', 'wmc-runtime', 'workqueue'],
+        'statics': ['src/javascript/WMCore/WebTools/Agent',
+                    'src/javascript/WMCore/WebTools/WMBS',
```

Comment:
These two "WMCore/WebTools/..." is something that I need to test.
Unless I'm mistaken, this corresponds to the old WMBS service monitoring code that was in place for the agents, given that we have deprecated/removed all that - python - code I think we no longer need it. But again, it has to be tested.

Reply:
Removed all 3 WebTools statics, testing it in a replay now. If the replay is fine, then we should be good with these changes.

Reply:
A replay testing these went fine and I didn't spot any issues (related to WebTools or any other).
It should be safe to merge this now.